src/operator/leaky_relu.cc

```diff
@@ -22,14 +22,16 @@ Operator *LeakyReLUProp::CreateOperator(Context ctx) const {
 DMLC_REGISTER_PARAMETER(LeakyReLUParam);
 
 MXNET_REGISTER_OP_PROPERTY(LeakyReLU, LeakyReLUProp)
-.describe(R"code(Leaky ReLu activation
+.describe(R"code(Rectified Linear Unit that allows a small gradient when the unit is not active.
```

Comment:
changed to 
"Applies Leaky rectified linear unit activation element-wise to the input.

Leaky ReLUs attempt to fix the "dying ReLU" problem by allowing a small `slope` 
when the input is negative and has a slope of one when input is positive."